src/components/CurrencyLogoBridge/index.tsx

```diff
@@ -0,0 +1,135 @@
+import { ChainId, Currency, Token, DXD, SWPR } from '@swapr/sdk'
+import React, { ReactNode, useMemo } from 'react'
+import Skeleton from 'react-loading-skeleton'
+import styled from 'styled-components'
+
+import EtherLogo from '../../assets/images/ether-logo.png'
+import XDAILogo from '../../assets/images/xdai-logo.png'
+import DXDLogo from '../../assets/svg/dxd.svg'
+import SWPRLogo from '../../assets/images/swpr-logo.png'
+import carrotListLogoUrl from '../../assets/images/carrot.png'
+import { useActiveWeb3React } from '../../hooks'
+import useHttpLocations from '../../hooks/useHttpLocations'
+import { WrappedTokenInfo } from '../../state/lists/wrapped-token-info'
+import Logo from '../Logo'
+import { useBridgeTokenInfo } from '../../services/EcoBridge/EcoBridge.hooks'
+
+const getTokenLogoURL = (address: string) =>
+  `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${address}/logo.png`
```

Comment:
The repo has folders for other EVM chain. Shouldn't we add `chainId` and map that?

```typescript
{ 
    1: 'ethereum',
    100: 'xdai',
    // ... more mappings
}
```